app/src/test/scala/org/alephium/explorer/GenCoreApi.scala

```diff
@@ -271,7 +272,7 @@ object GenCoreApi {
         height = height,
         chainFrom = previousBlocks.last.chainFrom,
         chainTo = previousBlocks.last.chainTo,
-        transactions = AVector.from(txs),
+        transactions = AVector.from(txs :+ coinbaseTx),
```

Comment:
the generated data was also leading to some flakyness, as the last tx in a block was not a coinbase one.